Google.Api.Generator/Utils/Typ.cs

```diff
@@ -59,7 +59,7 @@ private sealed class Array : Typ
 
         private sealed class FromManual : Typ
         {
-            public FromManual(string ns, string name, bool isEnum = false) => (Namespace, Name, IsEnum) = (ns, name, IsEnum);
+            public FromManual(string ns, string name, bool isEnum = false) => (Namespace, Name, IsEnum) = (ns, name, isEnum);
```

Comment:
Ouch. It really feels like the compiler should have warned you about that.

Reply:
Yes, I'm surprised it didn't warn.